provider/resources.go

```diff
@@ -2027,6 +2027,24 @@ func Provider() tfbridge.ProviderInfo {
 					Source: "iap_web_backend_service_iam.html.markdown",
 				},
 			},
+			"google_iap_web_region_backend_service_iam_binding": {
+				Tok: gcpResource(gcpIAP, "WebRegionBackendServiceIamBinding"),
+				Docs: &tfbridge.DocInfo{
+					Source: "iap_web_region_backend_service_iam.html.markdown",
+				},
+			},
+			"google_iap_web_region_backend_service_iam_member": {
+				Tok: gcpResource(gcpIAP, "WebRegionBackendServiceIamMember"),
+				Docs: &tfbridge.DocInfo{
+					Source: "iap_web_region_backend_service_iam.html.markdown",
+				},
+			},
+			"google_iap_web_region_backend_service_iam_policy": {
+				Tok: gcpResource(gcpIAP, "WebRegionBackendServiceIamPolicy"),
+				Docs: &tfbridge.DocInfo{
+					Source: "iap_web_region_backend_service_iam.html.markdown",
```

Comment:
Initially tried running the upgrade with flag `--allow-missing-docs`, which was incorrect.